include/bluetooth/gatt.h

```diff
@@ -549,12 +549,13 @@ ssize_t bt_gatt_attr_read_chrc(struct bt_conn *conn,
  *  @param _write Characteristic Attribute write callback.
  *  @param _value Characteristic Attribute value.
  */
-#define BT_GATT_CHARACTERISTIC(_uuid, _props, _perm, _read, _write, _value) \
-	BT_GATT_ATTRIBUTE(BT_UUID_GATT_CHRC, BT_GATT_PERM_READ,		\
-			  bt_gatt_attr_read_chrc, NULL,			\
-			  ((struct bt_gatt_chrc[]) { { .uuid = _uuid,	\
-						       .value_handle = 0U, \
-						       .properties = _props, } })), \
+#define BT_GATT_CHARACTERISTIC(_uuid, _props, _perm, _read, _write, _value)  \
+	BT_GATT_ATTRIBUTE(BT_UUID_GATT_CHRC, BT_GATT_PERM_READ,		     \
+			  bt_gatt_attr_read_chrc, NULL,			     \
+			  ((struct bt_gatt_chrc[]) { { .uuid = _uuid,	     \
+						       .value_handle = 0U,   \
+						       .properties = _props, \
+						   } })),                    \
```

Comment:
Since you're changing this could you also fix the use of spaces instead of tabs for alignment, i.e. the whitespace before the `\` characters should be spaces, not tabs.

Reply:
How did you see that there was tabs in there in GH?

Reply:
I try to highlight the whitespace and see how much the cursor jumps :)

Reply:
> How did you see that there was tabs in there in GH?

You do not see the tabs? :)

![image](https://user-images.githubusercontent.com/180017/76871966-253b6080-6842-11ea-9a67-d57f9f087b36.png)

Reply:
No :( Care to spill you secrets?